javascript/javascript2/week1/shortword.js

```diff
@@ -1,11 +1,20 @@
 const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"]; // returns 'ø'
-function findShortest(d) {
-  return d.reduce(function (prevWord, currWord) {
-    if (currWord.length < prevWord.length) {
-      console.log(currWord);
-      return currWord;
-    } else return prevWord;
-  });
-}
+function findShortest(danishWords) {
+  let shortLength = Infinity;
+  let shortest = "";
+
+  if (danishWords.length > 0) {
+    for (let i = 0; i < danishWords.length; i++) {
+      if (
+        typeof danishWords[i] === "string" &&
+        danishWords[i].length < shortLength
```

Comment:
You can do just this . No need for `shortLength ` variable
```suggestion
        danishWords[i].length < shortest.length
```